crates/flowctl/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "flowctl"
+version = "0.1.0"
+edition = "2021"
```

Comment:
I don't think we've ever really taken a position on that. I think everyone's always just defaulted to installing a relatively recent version and upgrading whenever we want to use new rust features. IMO that's probably fine for now, since nobody outside of us is really depending on our crates as libraries.